Make the steps for deploying to Meta Quest 2 (and other Android headsets) clearer

dsnopek opened the pull request:
This PR is attempting to specifically address the points of confusion from https://github.com/godotengine/godot/issues/78427

Namely:

- Making it extra clear that you need to follow most of the normal Android setup steps before doing the things on the OpenXR-specific page.
- Changing one of the screenshots to have the "Use Gradle build" checkbox checked (as described in the text instructions, but confusingly shown unchecked in the original screenshot)
- Specifically linking to the "One-click deploy" documentation, and making it clear that developers may need to perform some extra steps from their headset vendor's documentation in order to make it work. And I added a note and link for the Meta Quest in particular, since it's the most popular headset.

I also fixed some terminology that I think is incorrect: "export template" -> "export preset"

BastiaanOlij commented:
Looks good!

I would maybe add a screenshot of the dropdown and option to run, I could create one for you to use that shows both Quest and Pico in the menu to make clear why it is a dropdown.

dsnopek commented:
Thanks :-)

> I could create one for you to use that shows both Quest and Pico in the menu to make clear why it is a dropdown

Yeah, that'd be awesome! Personally, I only have various flavors of Quest

BastiaanOlij commented:
@dsnopek you'll need to crop it and probably convert it to webp, but hopefully this helps?

![Godot Android VR dropdown](https://github.com/godotengine/godot-docs/assets/1945449/693ab8a6-f974-4f40-8d8e-01983a2bd5d6)

dsnopek commented:
Thanks so much, @BastiaanOlij!

I cropped Bastiaan's screenshot and blurred the areas behind the dropdown (since they are pretty distracting), converted to webp and added to the new section about one-click deploy.

Hopefully, the blur is an OK way to do this? I could also just cut those parts out, but I worried that that could be confusing since there _is_ stuff there and might it make it hard for folks to find the right part of the screen.

skyace65 commented:
For a image like that where you want to direct people to a specific part you would add an outline instead of blurring the background. Like this:
[android_one_click_deploy.zip](https://github.com/godotengine/godot-docs/files/11900886/android_one_click_deploy.zip)

And if you don't mind please update your PR with that image.

dsnopek commented:
Sure! I've updated the PR, replacing the image with yours. Thanks for making it :-)

BastiaanOlij reviewed (approved):
lgtm, nice work!

BastiaanOlij commented on tutorials/xr/deploying_to_android.rst:
```diff
@@ -35,14 +44,15 @@ There is a subfolder called **plugins** containing the new files.
-Creating the export templates
------------------------------
-You will need to setup a separate export template for each device as each device will need its own loader included.
+Creating the export profiles
```
Worth changing over, I'm probably to blame here with my broken English :P

dsnopek replied:
Heh, sorry, no it was my mistake :-)

I've fixed it in my latest push

skyace65 commented:
Thanks!
